x/auth/types/test_utils.go

```diff
@@ -4,11 +4,13 @@ import (
 	"github.com/tendermint/tendermint/crypto"
 	"github.com/tendermint/tendermint/crypto/secp256k1"
 
+	"github.com/cosmos/cosmos-sdk/codec/testdata"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func NewTestMsg(addrs ...sdk.AccAddress) *sdk.TestMsg {
-	return sdk.NewTestMsg(addrs...)
+func NewTestMsg(addrs ...sdk.AccAddress) *testdata.TestMsg {
+	return testdata.NewTestMsg(addrs...)
 }
```

Comment:
I don't see the need for this func

Reply:
Yeah, it's pointless basically. It's a holdover from before, but will remove.